.github/workflows/scheduled_test.yaml

```diff
@@ -0,0 +1,36 @@
+name: Scheduled Test
+
+on:
+  schedule:
+    # Execute weekly on Monday UTC time. Offset time to avoid high load period on github servers.
+    - cron: "15 3 * * 1"
+
+jobs:
+  integration-test:
+    name: Integration tests (lxd)
+    uses: ./.github/workflows/integration_test.yaml
+  notify-on-failure:
+    name: Notify on failure
+    needs: [integration-test]
+    if: ${{ inputs.notify && failure() }}
+    runs-on: ubuntu-latest
+    env:
+      USERNAME: ${{ secrets.email_username }}
+      PASSWORD: ${{ secrets.email_password }}
+    steps:
+      - name: Check credential exists
+        if: ${{ env.USERNAME == null || env.PASSWORD == null }}
+        run: |
+          echo "Missing email username and password as input for this workflow."
+          exit 1
+      - name: Notify Failure
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          username: ${{ env.USERNAME }}
+          password: ${{ env.PASSWORD }}
```

Comment:
Currently, the email server is set to gmail.
The credential need to be set up in this repo.
The username is `email_username`, and password is `email_password`. See line 18, 19.

Reply:
Does it make sense to have this in one place and reference it from charm repos, i.e. put this in the operator-workflows?

Reply:
It should work, but I am not clear on the details of that repo yet.
I will be looking into operator-workflows for another story this week.

Reply:
I don't think the `on` part can be in a reusable workflow, only the jobs where you reference reusable workflow using `uses`

Reply:
Yes, I think the schedule will remain on every repo, though integration tests and sending notifications can be reused.

Reply:
Are the required secrets set in this repo? You should have maintainer access

Reply:
The required credential are not setup.
I looked into the setting tab in this repo, and the secrets options are not there.
![image](https://user-images.githubusercontent.com/43424755/203449212-f880c970-daa0-4fe6-9cfd-faff4598a4f8.png)

For repo under my personal account, I was able to set the secrets in the setting:
![image](https://user-images.githubusercontent.com/43424755/203449398-fac4a504-85d8-4273-b0b8-acba0aa5ed87.png)

Reply:
Ok, please send me the secrets I need to set and I'll make it happen 👍